deploy/eck-elasticsearch/values.yaml

```
@@ -32,31 +32,79 @@ annotations: {}
 # ref: https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-stack-monitoring.html
 #
 monitoring: {}
+  # metrics:
+  #   elasticsearchRefs:
+  #   - name: monitoring
+  #     namespace: observability 
+  # logs:
+  #   elasticsearchRefs:
+  #   - name: monitoring
+  #     namespace: observability
 
 # Control the Elasticsearch transport module used for internal communication between nodes.
 # ref: https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-transport-settings.html
 #
 transport: {}
+  # service:
+  #   metadata:
+  #     labels:
+  #       my-custom: label
+  #   spec:
+  #     type: LoadBalancer
+  # tls:
+  #   subjectAltNames:
+  #     - ip: 1.2.3.4
+  #     - dns: hulk.example.com
+  #   certificate:
+  #     secretName: custom-ca
 
 # Settings to control how Elasticsearch will be accessed.
 # ref: https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-accessing-elastic-services.html
 #
 http: {}
+  # service:
+  #   metadata:
+  #     labels:
+  #       my-custom: label
+  #   spec:
+  #     type: LoadBalancer
+  # tls:
+  #   subjectAltNames:
+  #     - ip: 1.2.3.4
+  #     - dns: hulk.example.com
+  #   certificate:
+  #     secretName: custom-ca
```

Compared to the `transport` service, the `http` service expects the `selfSignedCertificate` field above `tls`.
Also, do we want to show how to disable it?
```suggestion
  # tls:
  #   selfSignedCertificate:
  #     disabled: false
  #     subjectAltNames:
  #       - ip: 1.2.3.4
  #       - dns: hulk.example.com
  #   certificate:
  #     secretName: custom-ca
```